examples/misc.lean

```diff
@@ -6,16 +6,139 @@ def bar2 {α : Q(Sort u)} (a : Q($α)) : Q($a = $a) := q(by simp)
+                ((Expr.const `List.nil [Level.zero]).app (Expr.const `Lean.Level [])))).app
+          (((Expr.const `Lean.Expr.app []).app
+                (((Expr.const `Lean.Expr.app []).app
+                      (((Expr.const `Lean.Expr.app []).app
+                            (((Expr.const `Lean.Expr.const []).app
+                                  ((Expr.const `Lean.Name.mkStr1 []).app (Expr.lit (Literal.strVal "Eq")))).app
+                              ((((Expr.const `List.cons [Level.zero]).app (Expr.const `Lean.Level [])).app
+                                    ((Expr.const `Lean.Level.succ []).app (Expr.const `Lean.Level.zero []))).app
+                                ((Expr.const `List.nil [Level.zero]).app (Expr.const `Lean.Level []))))).app
+                        (((Expr.const `Lean.Expr.const []).app
+                              ((Expr.const `Lean.Name.mkStr1 []).app (Expr.lit (Literal.strVal "Nat")))).app
+                          ((Expr.const `List.nil [Level.zero]).app (Expr.const `Lean.Level []))))).app
+                  ((((Expr.const `Lean.ToExpr.toExpr [Level.zero]).app (Expr.const `Nat [])).app
+                        (Expr.const `Lean.instToExprNat [])).app
+                    ((((Expr.const `OfNat.ofNat [Level.zero]).app (Expr.const `Nat [])).app
+                          (Expr.lit (Literal.natVal 3))).app
+                      ((Expr.const `instOfNatNat []).app (Expr.lit (Literal.natVal 3))))))).app
+            ((((Expr.const `Lean.ToExpr.toExpr [Level.zero]).app (Expr.const `Nat [])).app
+                  (Expr.const `Lean.instToExprNat [])).app
+              ((((Expr.const `OfNat.ofNat [Level.zero]).app (Expr.const `Nat [])).app (Expr.lit (Literal.natVal 3))).app
+                ((Expr.const `instOfNatNat []).app (Expr.lit (Literal.natVal 3)))))))).app
+    (((Expr.const `Lean.Expr.app []).app
+          (((Expr.const `Lean.Expr.app []).app
+                (((Expr.const `Lean.Expr.const []).app
+                      ((Expr.const `Lean.Name.mkStr1 []).app (Expr.lit (Literal.strVal "eq_self")))).app
+                  ((((Expr.const `List.cons [Level.zero]).app (Expr.const `Lean.Level [])).app
+                        ((Expr.const `Lean.Level.succ []).app (Expr.const `Lean.Level.zero []))).app
+                    ((Expr.const `List.nil [Level.zero]).app (Expr.const `Lean.Level []))))).app
+            (((Expr.const `Lean.Expr.const []).app
+                  ((Expr.const `Lean.Name.mkStr1 []).app (Expr.lit (Literal.strVal "Nat")))).app
+              ((Expr.const `List.nil [Level.zero]).app (Expr.const `Lean.Level []))))).app
+      ((((Expr.const `Lean.ToExpr.toExpr [Level.zero]).app (Expr.const `Nat [])).app
+            (Expr.const `Lean.instToExprNat [])).app
+        ((((Expr.const `OfNat.ofNat [Level.zero]).app (Expr.const `Nat [])).app (Expr.lit (Literal.natVal 3))).app
+          ((Expr.const `instOfNatNat []).app (Expr.lit (Literal.natVal 3)))))))
+-/
+#guard_msgs in
 #eval foo' 3
```

Comment:
My guess is that many of these longer outputs should just become `run_meta` instead

Reply:
OK, I switched to using `#guard_msgs(warning, error)` instead.